projects/packages/videopress/src/client/admin/components/video-storage-meter/index.tsx

```diff
@@ -33,7 +33,7 @@ const VideoStorageMeter: React.FC< VideoStorageMeterProps > = ( {
 
 	const progress = used / total;
 	const progressLabel = `${ ( progress * 100 ).toFixed() }%`;
-	const totalLabel = filesize( total, { base: 2 } );
+	const totalLabel = filesize( total, { base: 10 } );
```

Comment:
I think the calculation will be off if we change the base, as `storageUsed` is in base 2. Maybe we'll need to ditch `filesize` here.

Reply:
Found it. So maybe we should change how we compute the `storageUsed` as well.

We know that the value is returned in `MB`, and we are multiplying it by `1024 * 1024` to get the byte count, but actually we may need to change it to `1000 * 1000`, given that `1 MB = 10^6 bytes`.

What do you think?

Reply:
Found the resolver here.
`videopress_storage_used` is in `MB`, so the `1024 * 1024` to get the number of bytes is correct.

I think the problem lies in the actual available space given by the plan, because if it is `1 TB`, the PR is correct, but if it is `1 TiB`, we'll be showing less than the actual available space.
Say a user has more then `1 TB` but less than `1 TiB`, the storage meter will go beyond `100%`.

If we need to show `1 TB` meaning `1 TiB` to be more "familiar", then we'll need to keep the math as is, but change the label.

Reply:
Yeah, maybe we are using TB just to use a more familiar label.

About the `MB` being `1024 * 1024`, isn't `1 MiB = 1024 * 1024 bytes` and `1MB = 1000 * 1000 bytes`?

We may need to confirm that the plan has `1 TB` and not `1 TiB`, and then adapt everything around it.

Reply:
Yeah, I meant to say that `videopress_storage_used` is in `MiB`, sorry.

Reply:
So we just need to make sure what's the real limit from the plan, and like you said:

- If it's `1 TB`, then the PR is ready
- If it's `1 TiB` and we use `TB` for familiarity, then we may need to go back to the `1024` multiples and just replace the label

Reply:
:chef-kiss:

Reply:
As far as I understood from here p7pQDF-6l3-p2, it's actually `1 TB` for real, so `1000 * 1000 * 1000 * 1000` bytes, so we are set on the PR changes so far.

Reply:
Just to consolidate the solution:

- the storage is 1 TB, so 1000 * 1000 * 1000 * 1000 bytes
- the storage used statistic uses MB, so 1000 * 1000 bytes